NavigationSystem/main_ASPire.cpp

```diff
@@ -0,0 +1,297 @@
+#include <string>
+#include "SystemServices/Logger.h"
+#include "MessageBus/MessageBus.h"
+
+#if SIMULATION == 1
+ #include "Simulation/SimulationNode.h"
+#else
+ #include "Hardwares/HMC6343Node.h"
+ #include "Hardwares/GPSDNode.h"
+ #include "Hardwares/ActuatorNodeASPire.h"
+ #include "Hardwares/CAN_Services/CANService.h"
+ #include "Hardwares/CANWindsensorNode.h"
+#endif
+
+#include "Navigation/WaypointMgrNode.h"
+#include "WorldState/StateEstimationNode.h"
+#include "WorldState/WindStateNode.h"
+#include "HTTPSync/HTTPSyncNode.h"
+#include "Xbee/XbeeSyncNode.h"
+#include "WorldState/VesselStateNode.h"
+
+#if LOCAL_NAVIGATION_MODULE == 1
+  #include "Navigation/LocalNavigationModule/LocalNavigationModule.h"
+  #include "Navigation/LocalNavigationModule/Voters/WaypointVoter.h"
+  #include "Navigation/LocalNavigationModule/Voters/WindVoter.h"
+  #include "Navigation/LocalNavigationModule/Voters/ChannelVoter.h"
+  #include "Navigation/LocalNavigationModule/Voters/ProximityVoter.h"
+  #include "Navigation/LocalNavigationModule/Voters/MidRangeVoter.h"
+#else
+  #include "Navigation/LineFollowNode.h"
+#endif
+#include "LowLevelControllers/LowLevelControllerNodeASPire.h"
+
+
+#if USE_OPENCV_COLOR_DETECTION == 1
+#include "WorldState/obstacledetection/colorDetectionNode.h"
+#endif
+
+#include "Messages/DataRequestMsg.h"
```

Comment:
`	// NOTE - Jordan: Just to ensure messages are following through the system
	MessagePtr dataRequest = std::make_unique<DataRequestMsg>(NodeID::MessageLogger);
	messageBus.sendMessage(std::move(dataRequest));`

It is used but it is not necessary! Check line 240 in main_ASPire.cpp